development/compile-assets.md

```diff
@@ -9,7 +9,17 @@ Some components in PrestaShop, like Javascript or SCSS files, need to be compile
 
 ## Requirements
 
-We use [Webpack](https://webpack.js.org/) to compile assets. You only need NodeJS from 8.x to 12.x ([get it here](https://nodejs.org/)), NPM will take care of it all.
+We use [Webpack](https://webpack.js.org/) to compile assets. PrestaShop is using NodeJS ([get it here](https://nodejs.org/)), NPM will take care of it all.
+
+Here is a list of compatibility: 
+
+| PrestaShop Versions | NodeJS Versions |
+|---------------------|-----------------|
+| 8.x                 | 14.x            |
+
+If you can't manage to compile assets, you may be using an older version of PrestaShop. We strongly recommend you ton install a package such as ['n'](https://www.npmjs.com/package/n?activeTab=versions) or [nvm](https://github.com/nvm-sh/nvm) to try with an older version of NodeJS.
+
+On Windows, you'll probably need to install `windows-build-tools` using `npm i --global windows-build-tools` in order to make [node-gyp](https://github.com/nodejs/node-gyp#on-windows) working.
```

Comment:
```suggestion
On Windows, you will probably need to install `windows-build-tools` using `npm i --global windows-build-tools` in order to make [node-gyp](https://github.com/nodejs/node-gyp#on-windows) working.
```
Maybe a little bit more formal 😉  ?

Reply:
What do you mean @matks ?

Reply:
Using "you'll" or "we'll" or "what's" is a bit friendly, a bit chill 😄 the kind of tone you use with friends. Maybe for the docs we should aim for a tone a little bit more boring, more formal, more business-oriented

This is why I suggest to use `you will` instead of `you'll`